indico/modules/events/editing/controllers/backend/util.py

```diff
@@ -16,9 +16,10 @@
 from indico.util.i18n import _
 
 
-def review_and_publish_editable(revision, action, comment, tags=frozenset(), files=None):
+def review_and_publish_editable(revision, action, comment, tags=frozenset(), files=None, management=False):
```

Comment:
I'd make at least the management argument kw-only:
```suggestion
def review_and_publish_editable(revision, action, comment, tags=frozenset(), files=None, *, management=False):
```